allure-grpc/src/main/resources/tpl/grpc-request.ftl

```diff
@@ -0,0 +1,33 @@
+<html>
+<#-- @ftlvariable name="data" type="io.qameta.allure.grpc.GrpcRequestAttachment" -->
+<head>
+    <meta http-equiv="content-type" content="text/html; charset = UTF-8">
+    <script src="https://yastatic.net/jquery/2.2.3/jquery.min.js" crossorigin="anonymous"></script>
+
+    <link href="https://yastatic.net/bootstrap/3.3.6/css/bootstrap.min.css" rel="stylesheet" crossorigin="anonymous">
+    <script src="https://yastatic.net/bootstrap/3.3.6/js/bootstrap.min.js" crossorigin="anonymous"></script>
+
+    <link type="text/css" href="https://yandex.st/highlightjs/8.0/styles/github.min.css" rel="stylesheet"/>
+    <script type="text/javascript" src="https://yandex.st/highlightjs/8.0/highlight.min.js"></script>
+    <script type="text/javascript" src="https://yandex.st/highlightjs/8.0/languages/json.min.js"></script>
+    <script type="text/javascript">hljs.initHighlightingOnLoad();</script>
+
+    <style>
+        pre {
+            white-space: pre-wrap;
+        }
+    </style>
+</head>
+<body>
+<div>
+    <pre><code>gRPC: <#if data.url??>${data.url}<#else>Unknown</#if></code></pre>
+</div>
+
+<#if data.body??>
+    <h4>Message body</h4>
```

Comment:
Wouldn't it be better to specify the body of which message? "Request body" seems to be more readable, same in the response template.